Description: fix unknown latex command in example.text
 causing non UTF-8 chars to appear in stdout with recent texlive versions
Bug-Debian: https://bugs.debian.org/1002115
Bug: https://github.com/gettalong/kramdown/issues/744
Author: Cédric Boutillier <boutil@debian.org>
Last-Update: 2022-01-29

--- a/test/testcases/man/example.text
+++ b/test/testcases/man/example.text
@@ -80,6 +80,6 @@
 
 *[MD]: Markdown
 
-Math elements work $$\l = 5$$ inline and in block form:
+Math elements work $$\lambda = 5$$ inline and in block form:
 
 $$\lambda_5 = \alpha + 4$$
--- a/test/testcases/man/example.man
+++ b/test/testcases/man/example.man
@@ -114,7 +114,7 @@
 .P
 Abbreviations like MD can be used but the abbreviation title is ignored\.
 .P
-Math elements work \fB\el = 5\fP inline and in block form:
+Math elements work \fB\elambda = 5\fP inline and in block form:
 .sp
 .RS 4
 .EX
